clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -0,0 +1,495 @@
+                    dst_context.fs.move(tmp_dst_name, dst_actual_path)
+                except BaseException:  # noqa
+                    with suppress(FileNotFoundError):
+                        dst_context.fs.delete_file(tmp_dst_name)
+                    raise
+        else:
+            raise ValueError(
+                "Storage location: `{}` doesn't support now.".format(
+                    src_context.actual_path
+                )
+            )
+
+    def mv(self, src, dst, **kwargs):
+        src_identifier: NameIdentifier = self._extract_identifier(src)
+        dst_identifier: NameIdentifier = self._extract_identifier(dst)
+        if not src_identifier == dst_identifier:
+            raise ValueError(
+                "Destination file path identifier: `{}` should be same with src file path identifier: `{}`.".format(
+                    dst_identifier, src_identifier
+                )
+            )
+        src_context: FilesetContext = self._get_fileset_context(src)
+        if src_context.actual_path.startswith(StorageType.HDFS.value):
+            if self._check_mount_single_file(src_context.fileset, src_context.fs):
+                raise ValueError(
+                    "Cannot cp file of the fileset: {} which only mounts to a single file.".format(
+                        src_identifier
+                    )
+                )
+            dst_context: FilesetContext = self._get_fileset_context(dst)
+
+            src_actual_path = self._strip_protocol(src_context.actual_path).rstrip("/")
+            dst_actual_path = self._strip_protocol(dst_context.actual_path).rstrip("/")
+
+            src_context.fs.move(src_actual_path, dst_actual_path)
+        else:
+            raise ValueError(
+                "Storage location: `{}` doesn't support now.".format(
+                    src_context.actual_path
+                )
+            )
+
+    def rm_file(self, path):
+        context: FilesetContext = self._get_fileset_context(path)
+        if context.actual_path.startswith(StorageType.HDFS.value):
+            actual_path = self._strip_protocol(context.actual_path)
+            context.fs.delete_file(actual_path)
+        else:
+            raise ValueError(
+                "Storage location: `{}` doesn't support now.".format(
+                    context.actual_path
+                )
+            )
+
+    def rm(self, path, recursive=False, **kwargs):
+        context: FilesetContext = self._get_fileset_context(path)
+        if context.actual_path.startswith(StorageType.HDFS.value):
+            from pyarrow.fs import FileType
+
+            actual_path = self._strip_protocol(context.actual_path).rstrip("/")
+            [info] = context.fs.get_file_info([actual_path])
+            if info.type is FileType.Directory:
+                if recursive:
+                    context.fs.delete_dir(actual_path)
+                else:
+                    raise ValueError("Can't delete directories without recursive=False")
```

Comment:
The message is wrong? should be "Cann't delete directory with recursive=False"?

Reply:
Fixed this.